Source/WebCore/platform/graphics/coretext/FontPlatformDataCoreText.cpp

```diff
@@ -70,18 +69,23 @@ static RetainPtr<CTFontDescriptorRef> findFontDescriptor(CFStringRef referenceUR
 
 RetainPtr<CTFontRef> createCTFont(CFDictionaryRef attributes, float size, CTFontDescriptorOptions options, CFStringRef referenceURL, CFStringRef desiredPostScriptName)
 {
-    auto fontDescriptor = adoptCF(CTFontDescriptorCreateWithAttributes(attributes));
+    auto desiredReferenceURL = adoptCF(static_cast<CFURLRef>(CFURLCreateWithString(kCFAllocatorDefault, referenceURL, nullptr)));
+
+    auto fontDescriptor = adoptCF(CTFontDescriptorCreateWithAttributesAndOptions(attributes, options));
     if (fontDescriptor) {
         auto font = adoptCF(CTFontCreateWithFontDescriptorAndOptions(fontDescriptor.get(), size, nullptr, options));
```

Comment:
Before, we were passing in the `CTFontDescriptorOptions` value to `CTFontCreateWithFontDescriptorAndOptions`, even though it's expecting a `CTFontOptions`. That seems wrong. Now, we're also passing the `CTFontDescriptorOptions` into the font descriptor creation call (which seems right, since that's what it's expecting). Should we be not passing this in to `CTFontCreateWithFontDescriptorAndOptions`?